src/IECorePython/TypeIdBinding.cpp

```diff
@@ -243,10 +243,6 @@ void bindTypeId()
 		.value( "ObjectVectorParameter", ObjectVectorParameterTypeId )
 		.value( "DateTimeData", DateTimeDataTypeId )
 		.value( "DateTimeParameter", DateTimeParameterTypeId )
-		.value( "TimeDurationData", TimeDurationDataTypeId )
-		.value( "TimeDurationParameter", TimeDurationParameterTypeId )
-		.value( "TimePeriodData", TimePeriodDataTypeId )
-		.value( "TimePeriodParameter", TimePeriodParameterTypeId )
```

Comment:
Since we're targeting Cortex 10.7 with this work, can we also remove the value from the C++ enum?

While on the topic of Cortex 10.7 - we need to target the PR to `main` since we can't make breaking changes on `RB-10.6`.